www/css/cytube.css

```diff
@@ -703,4 +703,4 @@ body.hd #resize-video-larger, body.hd #resize-video-smaller {
 
 .userlist-ignored {
     text-decoration: line-through;
-}
+}
```

Comment:
Can you try to remove what change this is?
When we merge cytube, it might cause 1 more merge conflict if its there

Reply:
this im not sure how to remove. it's a discrepancy in line endings i think but changing from `LF` (line feed) to `CRLF` (control ...something... line feed) creates a diff on the whole file:
![image](https://github.com/user-attachments/assets/4071936a-433b-4b17-be15-f9fa8678b57f)

If you got any suggestions im all ears. totally agree about minimizing changes to cytube files

Reply:
yeah don't worry about it.
Not sure what it is either, it might be able to be automerged anyway.
Just leave it.